### PR TITLE
Upgrade to scala 2.13

### PR DESCRIPTION
**Pull request by gheine**

### CODE REVIEW

File: src/main/scala/io/flow/stream/Controller.scala [hunk 1]
```diff
@@ -131,34 +131,32 @@ case class Controller() extends io.flow.build.Controller {
...
+        if (payloadTypes.isEmpty) {
+          println(s"Skipping non v2 upserted union ${apiBuilderUnion.qualified} member ${apiBuilderModel.qualified}: payload type not found")
+        }
+        for {
+          pt <- payloadTypes
+          fld <- payloadField.toSeq
+        } yield {
+          EventType.Upserted(apiBuilderModel.name, typeName, fld.name, pt.model, discriminator)
+        }
+      case UnionMemberRx(typeName, eventType, _) if eventType == "deleted" =>
```
Comment (benwaffle) [inline src/main/scala/io/flow/stream/Controller.scala:149]: are "upserted" and "deleted" the only two options?

File: build.sbt [hunk 2]
```diff
@@ -2,7 +2,7 @@ name := "api-build"
 
 organization := "io.flow"
 
-scalaVersion in ThisBuild := "2.12.10"
+scalaVersion in ThisBuild := "2.13.1"
```
Comment (benwaffle) [inline build.sbt:5]: let's do 2.13.3

File: src/main/scala/io/flow/stream/Controller.scala (hunk 1 above)
Comment (gheine) [reply src/main/scala/io/flow/stream/Controller.scala:149]: yes

File: build.sbt (hunk 2 above)
Comment (gheine) [reply build.sbt]: 👍

**Pull request merged**